Control coverage/Feature/ServiceBus.md

```diff
@@ -69,3 +69,57 @@ Get-Help Remove-AzServiceBusAuthorizationRule -full
 
 ___ 
 
+## Azure_ServiceBus_DP_Use_Secure_TLS_Version
+
+### Display Name
+Use approved version of TLS for Azure Service Bus Namespace.
+
+### Rationale
+TLS provides privacy and data integrity between client and server. Using approved TLS version significantly reduces risks from security design issues and security bugs that may be present in older versions.
+
+### Control Settings
+
+```json
+{
+    "MinReqTLSVersion": "1.2"
+}
+```
+
+### Control Spec
+
+> **Passed:**
+> (*If Minimum TLS version is set to 1.2 or higher)
+> TLS settings are properly configured.
+>
+> **Failed:**
+> (*If Minimum TLS version is set to 1.0 or 1.1)
+> Current minimum TLS version is {currentMinTLSVersionString} which is less than required version 1.2
+> 
+> **Error:** 
+> Required minimum TLS version is not set properly in control settings.
+>
+### Recommendation
+
+- **Azure Portal**
+
+  Refer https://learn.microsoft.com/en-us/azure/service-bus-messaging/transport-layer-security-configure-minimum-version#specify-the-minimum-tls-version-in-the-azure-portal
```

Comment:
Give recommendations steps to fix existing non-compliant resources.

Reply:
Hey Tarun, the same link points to how we can fix for existing resources as well. Adding snapshot below

![image](https://user-images.githubusercontent.com/30566748/203524803-9d50d6eb-cd6f-43c1-8557-58b5a436e1ac.png)